app/views/root/_ga_postcode_error_tracking.html.erb

```diff
@@ -1,9 +1,11 @@
-<script type="text/javascript">
-  GOVUK.analytics.trackEvent(
-    "userAlerts:<%= alert_type %>",
-    "postcodeErrorShown:<%= @location_error.postcode_error %>",
-    {
-      label: "<%= t(@location_error.message, @location_error.message_args) %>"
-    }
-  );
-</script>
+<% if @location_error && @location_error.postcode_error %>
+  <script type="text/javascript">
```

Comment:
`type="text/javascript"` is redundant here.

Can this use the `auto-track-event` module, and the values placed as attributes elsewhere?

Inline script tags are one of the reasons we can't adopt CSP https://en.wikipedia.org/wiki/Content_Security_Policy